OrbitQt/orbitwatchwidget.cpp

```diff
@@ -275,7 +275,7 @@ void OrbitWatchWidget::OnUpdateWatch(const Variable* a_Variable) {
 void OrbitWatchWidget::on_FindLineEdit_textChanged(const QString& /*arg1*/) {}
 
 //-----------------------------------------------------------------------------
-void OrbitWatchWidget::on_RefreshButton_clicked() { GOrbitApp->RefreshWatch(); }
+void OrbitWatchWidget::on_RefreshButton_clicked() {}
```

Comment:
It should be fine to remove this function entirely.

Reply:
Yep, done.